packages/snap/src/asset/index.ts

```diff
@@ -8,13 +8,29 @@ import {SnapConfig} from "@nodefactory/metamask-polkadot-types";
 
 export const POLKADOT_SNAP_ASSET_IDENTIFIER = "polkadot-snap-asset";
 
+function getCustomViewUrl(configuration: SnapConfig, address: string): string {
```

Comment:
do we even need that since tx are in state?

Reply:
@mpetrunic you are right, especially because it is possible to set this trough configuration.

Reply:
i dont think we even need explorer url in snap

Reply:
I am not sure where in the snap are you referring? Because after refactoring this, we won't be using explorer anywhere inside snap. 

The only usage is inside _example_ (to provide a link on the executed transaction on explorer)